### DISCUSSION TITLE
What is your favorite vegetable?

### DISCUSSION THREAD

**Discussion by take0x** [NAIST-SE/AttractingContributors#1056] [2024 Attraction Contributions]
I like tomatoes. What about you?
https://github.com/take0x/AttractingContributors-Vegetable

**Comment by 37integer** [comment]
白菜

**Comment by natsu9731** [comment]
tomato

**Comment by take0x** [comment]
Thank you for your comments. I would like you to make a pull request.

**Reply by gill876** [reply]
I really love cabbage!

Please contribute to my project!
https://github.com/gill876/AttractingContributors-Hiking
